test/Plutus/Time.purs

```diff
@@ -192,15 +191,15 @@ suite = do
       toFromAesonTest "EndTimeLessThanTime" $ EndTimeLessThanTime absTimeFixture
       toFromAesonTest "CannotGetBigIntFromNumber" CannotGetBigIntFromNumber
     group "PosixTimeToSlotError" do
-      toFromAesonTest "CannotConvertAbsSlotToSlot" posixTimeToSlotErrFixture
+      -- toFromAesonTest "CannotConvertAbsSlotToSlot" posixTimeToSlotErrFixture
       toFromAesonTest "PosixTimeBeforeSystemStart" $ PosixTimeBeforeSystemStart
         posixTimeFixture
       toFromAesonTest "StartTimeGreaterThanTime" $ StartTimeGreaterThanTime
         absTimeFixture
       toFromAesonTest "EndSlotLessThanSlotOrModNonZero" $
         EndSlotLessThanSlotOrModNonZero absSlotFixture modTimeFixture
-      toFromAesonTest "CannotConvertAbsSlotToSlot" $ CannotConvertAbsSlotToSlot
-        absSlotFixture
+      -- toFromAesonTest "CannotConvertAbsSlotToSlot" $ CannotConvertAbsSlotToSlot
+      --   absSlotFixture
```

Comment:
What's the reason for removing these tests?

Reply:
@vvtran I had originally planned to use `BigInt` for `Slot` and removed these tests because the conversion could never fail. Restored them, thanks for pointing this out.